interface/themes/oe-common/README-oe-common.md

```diff
@@ -0,0 +1,7 @@
+### Purpose
+
+Contains scss that is shared between all themes  
+These rules generally apply to either individual pages or are oe- rules to be used anywhere in the application  
+Here because they don't fit the category of **'core'** component  
+Can be used flexibly by importing individual scss files or by importing a single scss file called ```all-common-import.scss``` that has all the individual scss files  
+**REMEMBER!!** for ```all-common-import.scss``` to do what it says, each time a scss file is added to this directory an ```@ import <file>``` line has to be added to ```all-common-import.scss```  
```

Comment:
hi @zbig01 ,
I was about to bring this into the codebase when I realized that none of these all-common-import stuff were in any of the color css styling scripts. It looks like you removed this in one of your later commits. I am just checking to ensure you meant to do that.
thanks,
-brady

Reply:
My mistake I brought it in in my PR